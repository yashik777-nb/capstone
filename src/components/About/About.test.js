import React from "react";
import { configure } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { createMount } from "@material-ui/core/test-utils";

import About from "./About";

configure({ adapter: new Adapter() });

describe("About Page Tests", () => {
  let wrapper;

  beforeEach(() => {
    const render = createMount();
    wrapper = render(<About />);
  });

  it("Should render one h4", () => {
    expect(wrapper.find("h4").length).toEqual(1);
  });

  it("Should render one h6", () => {
    expect(wrapper.find("h6").length).toEqual(1);
  });

  test("Should Render About page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
