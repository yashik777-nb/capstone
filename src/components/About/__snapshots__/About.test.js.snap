// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Tests Should Render About page 1`] = `ReactWrapper {}`;
